core/src/main/kotlin/transformers/documentation/DefaultDocumentationNodeMerger.kt

```diff
@@ -7,6 +7,7 @@ import org.jetbrains.dokka.plugability.DokkaContext
 internal object DefaultDocumentationNodeMerger : DocumentationNodeMerger {
     override fun invoke(modules: Collection<Module>, context: DokkaContext): Module =
         Module(
+            modules.first().name,
```

Comment:
This should check whether all the modules have the same name and log an error if not

Reply:
Done.

Reply:
I think that the behavior of `all{}` on an empty list is unintuitive and we should remove `drop(1)` even though it  works correctly. WDYT @Kordyjan?

Reply:
Removed.